nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -130,42 +134,83 @@ where
                     .unwrap_or("".to_string())
                     .trim()
                     .to_string();
-                let model = asset::ActiveModel {
+
+                // Set initial mint info.
+                let asset_model = asset::ActiveModel {
                     id: Set(id_bytes.to_vec()),
-                    owner: Set(Some(owner.to_bytes().to_vec())),
                     owner_type: Set(OwnerType::Single),
-                    delegate: Set(delegate),
                     frozen: Set(false),
-                    supply: Set(1),
-                    supply_mint: Set(None),
-                    compressed: Set(true),
                     tree_id: Set(Some(bundle.keys.get(3).unwrap().0.to_vec())),
-                    specification_version: Set(SpecificationVersions::V1),
-                    specification_asset_class: Set(SpecificationAssetClass::Nft),
-                    nonce: Set(nonce as i64),
-                    leaf: Set(Some(le.leaf_hash.to_vec())),
+                    specification_version: Set(Some(SpecificationVersions::V1)),
+                    specification_asset_class: Set(Some(SpecificationAssetClass::Nft)),
+                    nonce: Set(Some(nonce as i64)),
                     royalty_target_type: Set(RoyaltyTargetType::Creators),
                     royalty_target: Set(None),
                     royalty_amount: Set(metadata.seller_fee_basis_points as i32), //basis points
                     asset_data: Set(Some(id_bytes.to_vec())),
-                    seq: Set(seq as i64), // gummyroll seq
-                    slot_updated: Set(slot_i),
+                    slot_updated: Set(Some(slot_i)),
                     data_hash: Set(Some(data_hash)),
                     creator_hash: Set(Some(creator_hash)),
                     ..Default::default()
                 };
 
-                // Do not attempt to modify any existing values:
-                // `ON CONFLICT ('id') DO NOTHING`.
-                let query = asset::Entity::insert(model)
+                // Upsert asset table base info.
+                let query = asset::Entity::insert(asset_model)
                     .on_conflict(
                         OnConflict::columns([asset::Column::Id])
-                            .do_nothing()
+                            .update_columns([
```

Comment:
question - if it's possible in the contract to see two mint_v1 for the same asset at different slots (?) then this could overwrite newer data with older data.

Reply:
It is not possible to have two `mint_v1` for the same `asset_id`, because in the contract, the `asset_id` is a PDA where one of the seeds is the `num_minted` for the specific tree.  Upon successful mint, `num_minted` is incremented.  It is monotonically increasing so we should never end up with the same `num_minted` for a given tree.